src/module.c

```diff
@@ -898,8 +900,7 @@ int TSDB_incrby(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 
     double incrby = 0;
     if (RMUtil_ParseArgs(argv, argc, 2, "d", &incrby) != REDISMODULE_OK)
-        return RedisModule_WrongArity(ctx);
-
+        return RedisModule_ReplyWithError(ctx, "TSDB: invalid increase/decrease value");
```

Comment:
Good catch, should be extracted to a different PR.

Reply:
PR #260